solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -276,6 +277,7 @@ public MiniSolrCloudCluster(
     Objects.requireNonNull(securityJson);
     this.baseDir = Objects.requireNonNull(baseDir);
     this.jettyConfig = Objects.requireNonNull(jettyConfig);
+    this.solrXml = solrXml == null ? DEFAULT_CLOUD_SOLR_XML : solrXml;
```

Comment:
Wondering if the similar logic at lines 491-493/492-494 can be removed now then?

Reply:
Yeah, I didn't see the other usages of this method with null. So I moved the defaulting here instead. Thanks for the review!